examples/table.rb

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+
+require 'rbbcc'
+include RbBCC
+
+b = BCC.new(text: <<CLANG)
+BPF_HASH(the_table_name, int);
+CLANG
+
+table = b.get_table('the_table_name', leaftype: 'int')
```

Comment:
without `leaftype` , the type will be parsed as `unsigned long long` from the BPF program, not sure how to fix this, but I will try to find the cause in future.

Reply:
Thanks! I'll check it out if I can!